assets/locales/ru/translation.json

```diff
@@ -223,23 +225,23 @@
   "Remote Path": "Удаленный путь",
   "BrowserStack Username": "Имя пользователя BrowserStack",
   "BrowserStack Access Key": "Ключ доступа BrowserStack",
-  "Bitbar API Key": "API ключ Bitbar",
-  "willUseCurrentlyRunningServer": "Will use currently-running Appium Desktop server",
+  "Bitbar API Key": "Ключ API Bitbar",
+  "willUseCurrentlyRunningServer": "Будет использован текущий сервер Appium Desktop",
   "Name": "Название",
   "enterYourSessionId": "Введите идентификатор сессии здесь",
-  "Proxy URL": "URL прокси сервера",
+  "Proxy URL": "URL прокси-сервера",
   "Source": "Источник",
   "Actions": "Действия",
   "Select Action Group": "Выберите группу действий",
   "Select Sub Group": "Выберите подгруппу",
   "Save Capability Set As": "Сохранить набор возможностей как…",
-  "Edit Raw JSON": "Редактировать JSON",
+  "Edit Raw JSON": "Изменить JSON",
   "Device": "Устройство",
   "Execute Mobile": "Выполнить на мобильном",
   "Execute": "Выполнить",
   "Android Activity": "Активность Android",
-  "Start Activity": "Начать действие",
-  "Current Activity": "Текущая активность",
+  "Start Activity": "Start Activity",
+  "Current Activity": "Текущий экран",
```

Comment:
I think it is better not to translate it as well, keep it as `en`

Reply:
All requests resolved. Thank you! There are several other fixes in the PR.